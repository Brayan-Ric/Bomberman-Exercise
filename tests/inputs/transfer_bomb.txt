S2 R F1
F2 _ F1
R B2 DU