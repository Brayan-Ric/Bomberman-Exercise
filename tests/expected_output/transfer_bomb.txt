_ R _
F1 _ F1
R B2 DU
